src/v/cluster/topics_frontend.cc

```diff
@@ -437,6 +457,16 @@ errc topics_frontend::validate_topic_configuration(
         }
     }
 
+    if (_features.local().should_sanction()) {
+        if (auto f = get_enterprise_feature(assignable_config); f.has_value()) {
+            vlog(
+              clusterlog.warn,
+              "An enterprise license is required to enable {}.",
+              f.value());
+            return errc::topic_invalid_config;
+        }
```

Comment:
Would it make sense to change the return type of `get_enterprise_feature` to `std::vector<std::string_view>` so we can report all the enterprise features being used?

Reply:
> Would it make sense to change the return type of `get_enterprise_feature` to `std::vector<std::string_view>` so we can report all the enterprise features being used?

Yeah, I like this.

Reply:
Done